plugins/kubernetes/app/views/kubernetes_releases/_create_form_step_1.html.erb

```diff
@@ -0,0 +1,115 @@
+<script type='text/ng-template' id='kubernetes/kubernetes_release_wizard_step_1.tmpl.html'>
+
+  <us-spinner spinner-theme="async-data-loader" ng-if="!wizardReady()">
+    <span class="spinner-label">Initializing wizard...</span>
+  </us-spinner>
+
+  <div ng-if="wizardReady() && empty(builds)">
+    <p class="no-builds-message text-warning">
+      You must create a build before creating a release.<br/>
+      Click <a ng-click="showProjectBuilds()">here</a> if you wish to create a build now.
+    </p>
+  </div>
+
+  <div ng-if="wizardReady()" class="form-container">
+    <form ng-submit="submit($event)">
+      <fieldset>
+        <div class="form-group clearfix">
+          <label for="build">
+            <span class="order">1.</span>
+            <span>Select the build:</span>
+          </label>
+
+          <div class="col-lg-4 no-padding">
+            <div id="environment-dropdown" class="btn-group" uib-dropdown uib-keyboard-nav>
+              <button type="button" class="btn btn-default" uib-dropdown-toggle>
+                {{ build.label || 'Choose build' }} <span class="caret"></span>
+              </button>
+              <ul class="uib-dropdown-menu" role="menu" aria-labelledby="environment-dropdown">
+                <li role="menuitem">
+                  <a ng-click="buildChanged()"></a>
+                </li>
+                <li role="menuitem" ng-repeat="build in builds">
+                  <a ng-click="buildChanged(build)">{{ build.label }}</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+
+        <div class="form-group clearfix">
+          <label for="environment">
+            <span class="order">2.</span>
+            <span>Select the environment:</span>
+          </label>
+
+          <div id="environment-dropdown" class="btn-group" uib-dropdown uib-keyboard-nav>
+            <button type="button" class="btn btn-default" uib-dropdown-toggle>
+              {{ environment.name || 'Choose environment' }} <span class="caret"></span>
```

Comment:
Humm, I confess I didn't even think about that scenario. Is it a valid scenario?

The other thing I'm not sure about is if AngularJS tries to resolve this when this section is not visible. This entire section will be displayed only if the `wizardReady()` check on the top most parent div succeeds. That check succeeds only if builds and environments were successfully loaded.

I'll give it a try, anyway.